fr.js

```diff
@@ -0,0 +1,128 @@
+l10n.translations.fr = {
+  logout: 'Déconnexion',
+  error: {
+    a: 'Il y a eu un petit soucis. Merci de',
+    b: 'recharger',
+    c: 'Voici quelques détails techniques :'
```

Comment:
Yes. In french, there is a space before two-elements' punctuation marks (? ! : ;). But i should replace it with a non-breakable one.